src/components/NFTArticle/elements/Figure.tsx

```diff
@@ -17,12 +17,22 @@ export function FigureElement({
 }: PropsWithChildren<Props>) {
   const selected = useSelected()
   const focused = useFocused()
+  const editor = useSlateStatic();
+
+  const handleFocus = () => {
+    const slateNode = ReactEditor.toSlateNode(editor, attributes.ref.current);
+    const path = ReactEditor.findPath(editor, slateNode)
+    const captionText = Node.string(slateNode)
+    const point = {path: [...path, 1], offset: captionText.length }
+    Transforms.select(editor, point)
+  }
```

Comment:
When a figure is focused. We want to restore selection to the caption inside the figure.